.travis.yml

```diff
@@ -17,13 +14,12 @@ before_install:
 - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 50
 - sudo apt-get install g++-4.8
 - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 50
-
 script: mvn install -DargLine="-Dlogback.configurationFile=src/test/resources/travis_logback.xml"
-
 after_success:
-  - test "${TRAVIS_PULL_REQUEST}" == "false" && test "${TRAVIS_TAG}" != "" && ./travis/publish_to_bintray.sh
-
+- test "${TRAVIS_PULL_REQUEST}" == "false" && test "${TRAVIS_TAG}" != "" && ./travis/publish_to_bintray.sh
+- ./aws/create_aws_ami.sh
```

Comment:
we should add this to the previous line or have similar test statements. we only want to deploy the ami image when we do official builds not for all PRs